test/integration/userBadges.test.js

```diff
@@ -0,0 +1,26 @@
+const chai = require('chai')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const app = require('../../server')
+
+chai.use(chaiHttp)
+
+describe('User badges', function () {
+  describe('GET /badges/:username', function () {
+    it('Should get the list of user badges', function (done) {
+      chai
+        .request(app)
+        .get('/badges/ankush')
+        .end((err, res) => {
+          if (err) { return done() }
```

Comment:
Test returns done() for all errors here... Shouldn't we be handling 404 , 401?

Reply:
Yes I'll add that.